README.md

```diff
@@ -33,6 +33,9 @@ Common use cases for Secrets Manager include:
 **Docker Runtime** | [Docs](https://docs.keeper.io/secrets-manager/secrets-manager/integrations/docker-runtime)| |
 **Kubernetes** | [Docs](https://docs.keeper.io/secrets-manager/secrets-manager/integrations/kubernetes)| |
 **GitLab** | [Docs](https://docs.keeper.io/secrets-manager/secrets-manager/integrations/gitlab-plugin)| |
+**XSOAR** | [Docs](https://docs.keeper.io/secrets-manager/secrets-manager/integrations/xsoar)| |
+**AWS HSM (Python)** | [Docs](https://docs.keeper.io/secrets-manager/secrets-manager/integrations/aws-kms)| |
```

Comment:
Can we add the Pypi artifacts to the right column for Entrust and AWS KMS?
https://pypi.org/project/keeper-secrets-manager-storage/